lib/index.js

```diff
@@ -43,6 +43,10 @@ export default function runContentfulImport (params) {
   infoTable.push([{colSpan: 2, content: 'The following entities are going to be imported:'}])
 
   Object.keys(options.content).forEach((type) => {
+    if (type === 'roles') {
```

Comment:
I would define a list of what the tool can import and filter through that in the `parseOptions`.